src/main/resources/css/settingsv2.css

```diff
@@ -0,0 +1,66 @@
+.settingsBorder{
+	
+	-fx-border-color: lightgrey;
```

Comment:
can you change the hover effect, that the background color gets a little brighter instead of the black border which appears currently? the border moves the menu-text currenlty which is not so nice

Reply:
done